src/main/java/org/gridsuite/modification/server/entities/equipment/modification/TwoWindingsTransformerModificationEntity.java

```diff
@@ -269,6 +269,22 @@ public class TwoWindingsTransformerModificationEntity extends BranchModification
     })
     private DoubleModificationEmbedded phaseTapChangerRegulationValue;
 
+    @Column(name = "phasetapchangerregulating")
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "phasetapchangerregulating")),
+        @AttributeOverride(name = "opType", column = @Column(name = "phasetapchangerregulatingOp"))
+    })
+    private BooleanModificationEmbedded phaseTapChangerRegulating;
+
+    @Column(name = "phasetapchangerloadtapchangingcapabilities")
+    @Embedded
+    @AttributeOverrides(value = {
+        @AttributeOverride(name = "value", column = @Column(name = "phasetapchangerloadtapchangingcapabilities")),
+        @AttributeOverride(name = "opType", column = @Column(name = "phasetapchangerloadtapchangingcapabilitiesOp"))
```

Comment:
For the naming : 
Shouldn't we put '_' in the names ? Examples : 
```suggestion
    @Column(name = "phase_tap_changer_regulating")
    @Embedded
    @AttributeOverrides(value = {
        @AttributeOverride(name = "value", column = @Column(name = "phase_tap_changer_regulating")),
        @AttributeOverride(name = "opType", column = @Column(name = "phase_tap_changer_regulating_op"))
    })
    private BooleanModificationEmbedded phaseTapChangerRegulating;

    @Column(name = "phase_tap_changer_load_tap_changing_capabilities")
    @Embedded
    @AttributeOverrides(value = {
        @AttributeOverride(name = "value", column = @Column(name = "phase_tap_changer_load_tap_changing_capabilities")),
        @AttributeOverride(name = "opType", column = @Column(name = "phase_tap_changer_load_tap_changing_capabilities_op"))
```